src/ResultWriter/faultoutput.f90

```diff
@@ -535,174 +541,179 @@ SUBROUTINE calc_FaultOutput( DynRup_output, DISC, EQN, MESH, MaterialVal, BND, t
-                ENDDO
-
-                !For calculating the 2d Jacobian matrix we will need the nodal coordinates
-                IF (MESH%Fault%Face(iFace,1,1) == 0) THEN
-                ! iElem is in the neighbor domain
-                ! The neighbor element belongs to a different MPI domain
-                iNeighbor           = MESH%Fault%Face(iFace,1,2)          ! iNeighbor denotes "-" side
-                iLocalNeighborSide  = MESH%Fault%Face(iFace,2,2)
-                iObject  = MESH%ELEM%BoundaryToObject(iLocalNeighborSide,iNeighbor)
-                MPIIndex = MESH%ELEM%MPINumber(iLocalNeighborSide,iNeighbor)
-                !
-                xV(1:4) = BND%ObjMPI(iObject)%NeighborCoords(1,1:4,MPIIndex)
-                yV(1:4) = BND%ObjMPI(iObject)%NeighborCoords(2,1:4,MPIIndex)
-                zV(1:4) = BND%ObjMPI(iObject)%NeighborCoords(3,1:4,MPIIndex)
-                ELSE
-                !
-                ! get vertices
-                xV(1:4) = MESH%VRTX%xyNode(1,MESH%ELEM%Vertex(1:4,iElem))
-                yV(1:4) = MESH%VRTX%xyNode(2,MESH%ELEM%Vertex(1:4,iElem))
-                zV(1:4) = MESH%VRTX%xyNode(3,MESH%ELEM%Vertex(1:4,iElem))
-                ENDIF
-
-                SELECT CASE(iSide)
-                 CASE(1)
-                    xab = (/ xV(3)-xV(1), yV(3)-yV(1), zV(3)-zV(1) /)
-                    xac = (/ xV(2)-xV(1), yV(2)-yV(1), zV(2)-zV(1) /)
-                 CASE(2)
-                    xab = (/ xV(2)-xV(1), yV(2)-yV(1), zV(2)-zV(1) /)
-                    xac = (/ xV(4)-xV(1), yV(4)-yV(1), zV(4)-zV(1) /)
-                 CASE(3)
-                    xab = (/ xV(4)-xV(1), yV(4)-yV(1), zV(4)-zV(1) /)
-                    xac = (/ xV(3)-xV(1), yV(3)-yV(1), zV(3)-zV(1) /)
-                 CASE(4)
-                    xab = (/ xV(3)-xV(2), yV(3)-yV(2), zV(3)-zV(2) /)
-                    xac = (/ xV(4)-xV(2), yV(4)-yV(2), zV(4)-zV(2) /)
-                ENDSELECT
-
-                grad2d(1,1) = dot_product(NormalVect_s(:),xab)
-                grad2d(1,2) = dot_product(NormalVect_t(:),xab)
-                grad2d(2,1) = dot_product(NormalVect_s(:),xac)
-                grad2d(2,2) = dot_product(NormalVect_t(:),xac)
-                CALL MatrixInverse2x2(JacobiT2d,grad2d)
-
-                dt_dx1 = JacobiT2d(1,1) * dt_dchi + JacobiT2d(1,2) * dt_dtau
-                dt_dy1 = JacobiT2d(2,1) * dt_dchi + JacobiT2d(2,2) * dt_dtau
-                Slowness = sqrt(dt_dx1**2 + dt_dy1**2)
-
-                IF (Slowness.EQ.0d0) THEN
-                  Vr=0d0
-                ElSE
-                  Vr=1d0/Slowness
-                ENDIF
-                DEALLOCATE(projected_RT)
-                iBndGP = DynRup_output%OutInt(iOutPoints,1)
-                ENDIF
-
-                OutVars = OutVars + 1
-                DynRup_output%OutVal(iOutPoints,1,OutVars)  = Vr
+            !calculation of the spatial derivatives of the rupture time
+            chi  = MESH%ELEM%BndGP_Tri(1,iBndGP)
```

Comment:
Where do all these changes come from? Please don't "just" change the formatting and indentation.

Reply:
that means I should redo this?